client/app/components/AccountConnectionForm.jsx

```diff
@@ -19,25 +19,32 @@ const formConfig = ({ t, fields, accountName }) => {
   }
 }
 
-const AccountConfigForm = ({ t, fields, dirty, submit, submitting }) => (
-  <div class='account-form'>
+const AccountConnectionForm = ({ t, fields, dirty, error, submit, submitting }) => (
+  <div class={'account-form' + (error === 'bad credentials' ? ' error' : '')}>
```

Comment:
Is it possible to just check the error variable presence here instead of the string content in order to be more extensible? At least just for the form class definition here. We should never have something in the error variable if all goes fine.